spec/features/cart/add_spec.rb

```diff
@@ -37,12 +37,12 @@
     it "I can't add a deactivated item" do
       user = create(:merchant_user, merchant: @mike)
       allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
-      
-      expect(@pen.active?).to eq(false)
+
       visit "/items/#{@pen.id}"
+      @pen.update(active?: false)
```

Comment:
this test was modified because visiting a show page for an inactive error now throws a 404